i18n/english.yml

```diff
@@ -87,6 +87,9 @@ components:
         placeholder: Additional information (optional)
       publishNewVersion:
         label: Publish snapshot as new feed version
+      publishProprietaryFiles: 
+        helpText: Proprietary files allow you to maintain certain proprietary Datatools features across feed version, e.g. Pattern names.
```

Comment:
```suggestion
        helpText: Proprietary files allow you to maintain certain Datatools features when re-importing, such as pattern names.
```

Cleanup langauge

Reply:
Good call, updated in https://github.com/ibi-group/datatools-ui/pull/991/commits/0f88ac57e5493d4bf7d7808169347b6d8ea9a788